### PR TITLE
[TASK] Rename dist files

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Resolves: #68

### CODE REVIEW

**Review by oliverklee — CHANGES_REQUESTED** [review]
There's also `.php_cs.dist ` (a PHP file) to rename.

File: composer.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
     },
     "scripts": {
         "ci:php:lint": "find *.php Classes Configuration Tests -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l",
-        "ci:php:codestyle": "php-cs-fixer fix --config .php_cs.dist -v --dry-run --using-cache false --diff --diff-format=udiff",
+        "ci:php:codestyle": "php-cs-fixer fix --config .php_cs.xml -v --dry-run --using-cache false --diff --diff-format=udiff",
```
Comment (oliverklee) [inline composer.json:71]: This file is PHP, not XML. Hence, the name needs to be `.php_cs.php`.

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (lukaszuznanski) [reply composer.json]: Sorry, late night changes :) fix pushed

**Review by oliverklee — CHANGES_REQUESTED** [review]
We'll also need to update the corresponding lines in the `.gitattributes` file. (I'm sorry that I missed that in my earlier reviews.)

**Review by oliverklee — APPROVED** [review]

**Pull request merged**